crates/bevy_animation/src/lib.rs

```diff
@@ -199,6 +215,8 @@ pub fn animation_player(
             }
             if elapsed < 0.0 {
                 elapsed += animation_clip.duration;
+            } else if elapsed > animation_clip.duration {
+                player.finished = true;
```

Comment:
Shouldn't this information (`player.finished`) be used to avoid calculating the `elapsed` time over and over again?

like add it together if `paused` check:
```rust
if (player.paused && !player.is_changed()) || player.finished {
    continue;
}
```

Reply:
Well I think:
```
if (player.paused || player.finished) && !player.is_changed() {
    continue;
}
```

would be more correct with this implementation, because some field like `repeat` could have changed.